python_scripts/api_requests.py

```diff
@@ -47,20 +68,44 @@ def get_payload(session, url, parameters):
     return resp.json()
 
 
-def scrape_days_from_api(start_date, end_date):
+def scrape_days_from_api(start_date, end_date, db_version, env, skip_write_s3 = True):
```

Comment:
Added new arguments - `db_version` and `env` are used together to specify path of files under database (e.g. v2 for dev or prod).

skip_write was for testing to stop overwriting s3 files